api.go

```diff
@@ -1203,7 +1213,12 @@ func (api *API) TranslateKeys(body io.Reader) ([]byte, error) {
 }
 
 type serverInfo struct {
-	ShardWidth uint64 `json:"shardWidth"`
+	ShardWidth       uint64 `json:"shardWidth"`
+	Memory           uint64 `json:"memory"`
+	CPUType          string `json:"cpuType"`
+	CPUPhysicalCores int
```

Comment:
Any reason to omit the `json` tags here? As it is, the case of the last three aren't consistent.

Reply:
Is "inattention" a "reason"?

And yeah, that should be fixed.

Reply:
On the cores: I read the comments for Linux, which said I'd get one entry per core, with "cores" = 1. Apparently on the Mac, it will come back with a single thing listing more cores... And what's bugging me is that I'm pretty sure the i5 is dual-core in physical cores, and has hyperthreading, so it *should* be claiming 4 "threads" (aka logical cores). Huh. Welp, back to the drawing board.

Reply:
weirder still: "2900" is the *expected* value for MHz (or the one I'd have expected). on linux i think this one would report 3300.